dev/integration_tests/ui/test_driver/driver_test.dart

```diff
@@ -3,8 +3,10 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io' show Platform;
```

Comment:
Cirrus is complaining that this import and the other new one below is unused. Remove?